cmd/util.go

```diff
@@ -291,12 +291,16 @@ func getKubeVersion() (*KubeVersion, error) {
 	_, err := exec.LookPath("kubectl")
 
 	if err != nil {
+		glog.V(3).Infof("Error locate kubectl %v", err)
 		_, err = exec.LookPath("kubelet")
 		if err != nil {
+			glog.V(3).Infof("Error locate kubelet %v", err)
 			// Search for the kubelet binary all over the filesystem and run the first match to get the kubernetes version
+			glog.V(3).Info("Search for kubelet binary all over filesystem")
```

Comment:
Personally I think this is unnecessary because the previous line is also a log, so you can see where you are in the code already